routes/cube/index.js

```diff
@@ -1418,9 +1418,14 @@ router.post('/edit/:id', ensureAuth, async (req, res) => {
   try {
     let cube = await Cube.findOne(buildIdQuery(req.params.id));
 
+    let viewQuery = 'table';
```

Comment:
These statements can be condensed a bit. The if statement isn't actually necessary
```suggestion
    const viewQuery = req.query.view || 'table';
```